lib/vagrant-libvirt/cap/synced_folder_virtiofs.rb

```diff
@@ -0,0 +1,110 @@
+require 'log4r'
+require 'ostruct'
+require 'nokogiri'
+require 'digest/md5'
+
+require 'vagrant/util/subprocess'
+require 'vagrant/errors'
+require 'vagrant-libvirt/errors'
+# require_relative "helper"
```

Comment:
If an update is needed, could remove this line

Reply:
Removed